MonoGame.Framework/PreparingDeviceSettingsEventArgs.cs

```diff
@@ -1,29 +1,29 @@
 // MonoGame - Copyright (C) MonoGame Foundation, Inc
 // This file is subject to the terms and conditions defined in
 // file 'LICENSE.txt', which is part of this source code package.
-
-using System;
-
-namespace Microsoft.Xna.Framework
-{
+
+using System;

```

Comment:
Nothing appears to have changed in this file other than formatting, we have strict guidance about NOT updating files just to apply formatting (because everyone's' formatters are different,) please revert.